_posts/2017/2017-11-26-choose-uuids-for-model-ids-in-rails.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Choose UUIDs for model ids in Rails'
+description: 'A little more complexity, a little more security'
+layout: article
+category: ruby
+image:
+  base: '2017/choose-uuids-for-model-ids-in-rails'
+  alt: 'Rainbow UUIDs'
+---
+
+Ruby on Rails has had the [ability](https://github.com/rails/rails/pull/21762) to use UUIDs ([Universally Unique IDentifiers](https://en.wikipedia.org/wiki/Universally_unique_identifier)) for ActiveRecord models since version 5.0.
+
+These are a native column type in PostgreSQL, there's more details on native Postgres types in the [Rails Guides](http://guides.rubyonrails.org/active_record_postgresql.html).
```

Comment:
Start this paragraph with: "UUIDs are..." Then try and say very briefly what they are...in case nobody knows what they are. Then go into "They are a native column type in PostgreSQL. You can find more information on native Postgres types in the Rails Guides." (Note the full-stop after 'PostgreSQL'.)

Ahh..I see you do link to the Wikipedia page...but I think it's worth spelling it out yourself.